### ISSUE TITLE
Windows phone 10

### ISSUE THREAD

**Issue by RuiOrey** [borismus/webvr-boilerplate#141]
Hi,

my windows phone is not working with the actual webvr boilerplate 
The problem is that the camera is always looking down and unresponsive of any phone movements and the camera is always looking down.
The application i'm using works on desktop and other mobiles (ios,android)
The sensors work on older versions of the boilerplate.

Lumia 640 windows phone 10

Thanks and great work!

**Comment by borismus** [comment]
Haven't tested on win phone at all, don't have one readily available. Could
you try to dig in and figure out what the problem might be? Maybe lack of
DeviceMotion support?

**Comment by RuiOrey** [comment]
I can try ... 
Well it works in [this old prototype](http://vrtest.ruidorey.webfactional.com/). 
The way the sensors are nowdays used/accessed is different from that old one?

**Comment by pindiespace** [comment]
I had a similar problem in Edge a few months ago - it seemed the camera was auto-rotated around 180 degrees by default.

**Comment by RuiOrey** [comment]
Did a simple test using the following 
- [this jsfiddle](http://jsfiddle.net/jdias/SrxtE/);
- [this CodePen example](http://codepen.io/verticalgrain/pen/LGbXVz);
- [Discovering Gale Crater](http://graphics.latimes.com/mars-gale-crater-vr/) 

and the Sensors seem to work... strange..

I will check again the basic webvr-boilerplate cube example (from where I derived the code of the application that is not working ONLY on my wp)...

**Comment by RuiOrey** [comment]
Yes.. on the basic cube example that comes with the project it also does keeps looking down and only rotates a bit if using touch...

**Comment by RuiOrey** [comment]
well I've been testing and seems at least my Lumia 640 does not support DeviceMotion's rotationRate (it supports timeStamp, accelerationIncludingGravity and acceleration). 
It seemed to work with deviceorientation.

**Comment by borismus** [comment]
We use rotationRate for motion prediction and other things. You could try
hacking together a fix, and I'd accept a patch. Unfortunately win phone
support isn't high on my priority list.

**Comment by RuiOrey** [comment]
I can try.. will take a look...

**Comment by RuiOrey** [comment]
For now only got to disable the rotationRate influence on windows phone. For having it probably the way to go is to re-use the device orientation old code..

**Comment by Rahman2326** [comment]
My Experience:
💗. Recovering Hacked WordPress Website
💗. Full Vulnerability Scan and WordPress website malware removal
💗. Vulnerability Assessment
💗. Antispam protection
💗. Analysis of database servers
💗. Restore a hacked website
💗. Ethical Hacking(White Hat Hacker)
💗. Hacked account recovery
💗. Password recovery
💗. Decode encrypted files
💗. Vulnerability Testing
💗....my WhatsApp number +8801314956025
